contracts/Pool.sol

```diff
@@ -28,3 +28,80 @@ contract Pool {
         receiver.transfer(value);
     }
 }
+
+struct ReceiverWeight {
+    address next;
+    uint32 weight;
+    bool isAttached;
+}
+
+/// @notice Helper methods for receiver weights list.
+/// The list works optimally if after applying a series of changes it's iterated over.
+/// The list uses 1 word of storage per receiver with a non-zero weight.
```

Comment:
Based on Solidity docs I think that it's true, but I'm not sure how to prove that.